Documentation/tutorial/docfx.exe_user_manual.md

```diff
@@ -277,6 +278,70 @@ Please refer to [How to Create Custom Templates](howto_create_custom_template.md
 }
 ```
 
+#### 3.2.2 Properties for docset config
+
+*Docsets* are used to build multiple docsets with a single run of DocFX. The behavior is like running DocFX multiple times with different `docfx.json`. It's useful when building multiple versions of the same repository, which will lead to `uid` collusion if not put in different docsets.
+**NOTE** Cross Reference between different docsets is not allowed.
```

Comment:
I tried and found it's appending to previous line, add one line before to start a new line?